src/definitions/modules/progress.js

```diff
@@ -606,7 +606,7 @@
                         }
                     },
                     barLabel: function (text) {
-                        $progresses.map(function (index, element) {
+                        $progresses.each(function (index, element) {
```

Comment:
this fix of `array-callback-return` rule should be ok (please note this is valid for `$().each(...` and not `$.each(...` in general, as the later has args swapped)

there are 4 more lines with errors:

```
src\definitions\modules\calendar.js
  1106:221  error  Array.prototype.some() expects a value to be returned at the end of function  array-callback-return
  1140:80   error  Array.prototype.some() expects a value to be returned at the end of function  array-callback-return
  1156:70   error  Array.prototype.some() expects a return value from function                   array-callback-return
  1182:101  error  Array.prototype.some() expects a value to be returned at the end of function  array-callback-return
```

currently `.some()` always returns false, which is probably not desired

@lubber-de would you be please so king and help with fixing these 4 lines?

Reply:
Thank you. Let's merge this PR, I can then finish #2589.